.travis.yml

```diff
@@ -21,7 +20,7 @@ before_install:
   - echo 'opcache.enable_cli=1' >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini
   - travis_retry composer self-update
 
-install: travis_retry composer update --optimize-autoloader --prefer-dist --prefer-stable --no-progress --no-interaction -vv $COMPOSER_FLAGS
+install: travis_retry composer install --optimize-autoloader --prefer-dist --no-progress --no-interaction -vv $COMPOSER_FLAGS
```

Comment:
I would prefer to insert the modified line directly into coding style stage.

Reply:
I can certainly try that, I just wonder if a composer.lock file should be committed at all for this project if we're concerned about testing against different versions of packages (as a library would typically do).

With the lock file you're basically saying that this application works against this set of dependencies, and has been tested that way, and should use those when installed, but if we're testing against multiple PHP versions which could pull different dependencies (and if one developer does a composer update under one PHP version, but another does one with another PHP version, the lock file could potentially be different between the two versions) then I think perhaps the lock file isn't serving its purpose here, and shouldn't be committed.

IMO, this project **should** be considered an application, which should only run on a particular version of PHP and have a lock file that defines which dependencies are known to work, and should be installed with it.  However, I know that @asgrim does use some of this project in the browscap site as a library. So perhaps it should be considered a library, and we ignore the lock file?

If that's the route we take, then we'll just have to react to these sorts of build failures that are caused by a dependency updating from one day to another, and either fix the code to work with that dependency (but still work with the "lowest"), or change `composer.json`  to lock to a specific version, or range of versions that don't have an issue.

Thoughts?

I can modify this PR to just remove and ignore the lock file, and lock the php-cs-fixer version at 2.7.1 so the build doesn't break until the issue with 2.7.2 can be addressed.